gcache/src/GCache_seqno.cpp

```diff
@@ -308,7 +308,13 @@ namespace gcache
 
         seqno_locked_count++;
 
-        if (seqno_g < seqno_locked) seqno_locked = seqno_g;
+        if (seqno_g < seqno_locked)
```

Comment:
On the assumption that multiple threads are calling this, there isn't memory barriers or mutexes to ensure the order of operations here.

Also applies to other operations on `seqno_{un,}lock` or accessing seqno_g directly.

Reply:
See line 300 there is mutex gu::Lock lock(mtx);.